frontend/src/app/authentication/service/token.service.ts

```diff
@@ -0,0 +1,33 @@
+import {Injectable} from '@angular/core';
+import {JWT} from "../../../../../api/specification/types";
+
+@Injectable({
+    providedIn: 'root'
+})
+export class TokenService {
+
+    constructor() {
+    }
+
+    isAuthenticated() {
+        console.log(localStorage.getItem("jwt"));
+        return localStorage.getItem("jwt") != null
+    }
+
+    deleteToken() {
+        localStorage.removeItem("jwt");
+    }
+
+    setToken(token: JWT) {
+        localStorage.setItem("jwt", token);
+    }
+
+    getUsername() {
+
+    }
```

Comment:
I don't know if it's a good idea to handle the username by the token service. The username is not part of the token and this will also lead to issues if we would allow users to change their username.

Reply:
I'd agree that it should probably be a different service. It probably makes sense to create a `UserService` or something similar.